tests/common/common_python_vec.py

```diff
@@ -333,12 +377,334 @@ def write_source_file(test_str, func_calls, test_name, input_file, output_file,
         source = source_file.read()
 
     source = replace_string_in_source_string(source,
-                                             type_str.replace(
-                                                 'cl::sycl::', '').replace(
-                                                     ' ', '_'), '$TYPE_NAME')
+                                             clear_type_str(type_str), '$TYPE_NAME')
     source = replace_string_in_source_string(source, test_name, '$CATEGORY')
     source = replace_string_in_source_string(source, test_str, '$TEST_FUNCS')
     source = replace_string_in_source_string(source, func_calls, '$FUNC_CALLS')
 
+    source = get_ifdef_string(source, type_str)
+
     with open(output_file, 'w+') as output:
         output.write(source)
+
+def get_standard_and_fixed_width_types():
+    types = list()
+    types.append('char')
+    types.append('cl::sycl::byte')
```

Comment:
Curious. Why not just creating the list with these 2 values?

Reply:
Yes, this is better, thank you. Changed to creating a list with them.